ec2/vpc.go

```diff
@@ -28,3 +29,18 @@ func (e *Ec2) ListVPCs(ctx context.Context) ([]map[string]string, error) {
 
 	return vpcs, nil
 }
+
+func (e *Ec2) GetVPCsByID(ctx context.Context, id string) (*ec2.Vpc, error) {
+	out, err := e.Service.DescribeVpcsWithContext(ctx, &ec2.DescribeVpcsInput{
+		Filters: []*ec2.Filter{
+			withVPCID(id),
+		},
+	})
+	if err != nil {
+		return nil, ErrCode("describing vpc", err)
+	}
```

Comment:
In all these functions we usually have a Debug statement here to log the `out`

Reply:
done.